### PR TITLE
fix link

### PR DESCRIPTION
**Pull request by rashedmyt**

### CODE REVIEW

File: src/docs/release/breaking-changes/index.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ description: A list of migration guides for breaking changes in Flutter.
 ---
 
 As described in our [breaking change
-policy][../../resources/compatibility], on occasion we publish guides
+policy](https://flutter.dev/docs/resources/compatibility), on occasion we publish guides
```
Comment (dnfield) [inline src/docs/release/breaking-changes/index.md]: I think you just want `(../../resources/compatibility)` here. This site gets deployed to other domains as well for staging/testing/etc.

Comment (dnfield) [reply src/docs/release/breaking-changes/index.md]: (so really the change should just be to fix the invalid markdown - just leave the URI alone)

Comment (rashedmyt) [reply src/docs/release/breaking-changes/index.md]: I saw [here](https://flutter.dev/docs/resources/compatibility#announcements-and-migration-guides), it used complete URL, so I thought of changing it. Will revert to as it was

**Comment by rashedmyt** [comment]
@dnfield Should I revert to complete URL? The build was passing with complete URL

**Comment by rashedmyt** [comment]
fixed in #3378

**Pull request closed without merging**